CI/check-services.py

```diff
@@ -323,7 +323,7 @@ def main():
             service_authors = dict()
 
         # set GitHub outputs
-        print(f'::set-output name=make_pr::true')
+        print(f'make_pr=true') >> $GITHUB_OUTPUT
```

Comment:
This is not valid python, annoyingly this change makes it more difficult here. You'd have to write everything to a file.

Perhaps adding a helper for that would work, e.g.
```
def set_output(name, value):
    try:
        with open(os.environ['GITHUB_OUTPUT'], 'a', newline='\n') as f:
            f.write(f'{name}={value}')
    except Exception as e:
        print(f'Writing to github output files failed: {e!r}')
```
And then replace these calls with `set_output('make_pr', 'true')` etc.

Reply:
Removed the file from this PR to make it less troublesome. Rodney will revisit this point sometime later.